core/src/main/scala/org/apache/spark/memory/package.scala

```diff
@@ -40,25 +40,26 @@ package org.apache.spark
- *       +-------------+         +-------------------+    |     |  +------------------+  |
- *                               | TaskMemoryManager |----+     |                        |
- *                               +-------------------+    |     |  +------------------+  |
- *                                                        +---->|  |OnHeapExecMemPool |  |
- *                                        *               |     |  +------------------+  |
- *                                        *               |     |                        |
- *       +-------------+                  *               |     |  +------------------+  |
- *       | MemConsumer |----+                             |     |  |OffHeapExecMemPool|  |
- *       +-------------+    |    +-------------------+    |     |  +------------------+  |
- *                          +--->| TaskMemoryManager |----+     |                        |
- *                               +-------------------+          +------------------------+
- * }}}
+ * {{{                                                          +---------------------------+
+ *       +-------------+                                        |       MemoryManager       |
+ *       | MemConsumer |----+                                   |                           |
+ *       +-------------+    |    +-------------------+          |  +---------------------+  |
+ *                          +--->| TaskMemoryManager |----+     |  |OnHeapStorageMemPool |  |
+ *       +-------------+    |    +-------------------+    |     |  +---------------------+  |
+ *       | MemConsumer |----+                             |     |                           |
+ *       +-------------+         +-------------------+    |     |  +---------------------+  |
+ *                               | TaskMemoryManager |----+     |  |OffHeapStorageMemPool|  |
+ *                               +-------------------+    |     |  +---------------------+  |
+ *                                                        +---->|                           |
+ *                                        *               |     |  +---------------------+  |
+ *                                        *               |     |  |OnHeapExecMemPool    |  |
+ *       +-------------+                  *               |     |  +---------------------+  |
+ *       | MemConsumer |----+                             |     |                           |
+ *       +-------------+    |    +-------------------+    |     |  +---------------------+  |
+ *                          +--->| TaskMemoryManager |----+     |  |OffHeapExecMemPool   |  |
+ *                               +-------------------+          |  +---------------------+  |
+ *                                                              |                           |
+ * }}}                                                          +---------------------------+
```

Comment:
I think the open and close braces have to be on their own lines. Here, the last part of the table is outside the block.

Reply:
Fixed